README.md

```diff
@@ -8,31 +8,43 @@ It is currently used in production all over the place in [Artsy.net](http://gith
 
 ### Meta
 
-* __State:__ production
-* __Production:__ [metaphysics-production.artsy.net](https://metaphysics-production.artsy.net/)
-* __Staging:__ [metaphysics-staging.artsy.net](https://metaphysics-staging.artsy.net/)
-* __CI:__ [Semaphore](https://semaphoreapp.com/artsy-it/metaphysics/)
-* __Point People:__ [@alloy](https://github.com/alloy) & [@mzikherman](https://github.com/mzikherman)
+-   **State:** production
+-   **Production:** [metaphysics-production.artsy.net](https://metaphysics-production.artsy.net/)
+-   **Staging:** [metaphysics-staging.artsy.net](https://metaphysics-staging.artsy.net/)
+-   **CI:** [Semaphore](https://semaphoreapp.com/artsy-it/metaphysics/)
+-   **Point People:** [@alloy](https://github.com/alloy) & [@mzikherman](https://github.com/mzikherman)
 
 ### Getting Setup
 
 Set up your `.env` file based on our example `.env.test` (the variables you must change for development are separated at the bottom).
 
 Install memcached and Yarn if you don't already have them:
-``` sh
+
+```sh
 brew install memcached
 npm install -g yarn
 ```
 
 To start up a development server, clone this repo and run:
 
-``` sh
+```sh
 yarn install
 npm run dev
 ```
 
 If you are using VS Code, you can read [debugging with VS Code](docs/debugging_with_vscode.md) to get inline debugging.
 
+### Setting up your local GraphiQL
+
+You will need to set up headers with both:
+
+-   `x-access-token` - Get your clientID and secret, use this command to generate a token.
```

Comment:
All I did with this was try to reduce the duplication of documentation across PS repos and instead link here: https://github.com/artsy/gravity/blob/master/doc/ApiAuthentication.md#create-xapp-token